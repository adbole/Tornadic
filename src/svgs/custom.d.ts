declare module "*.svg";
declare module "radar/*.svg";
declare module "conditions/*.svg"
declare module "widget/*.svg"