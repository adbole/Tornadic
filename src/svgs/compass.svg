<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   id="compass"
   width="240"
   height="240"
   version="1.1"
   sodipodi:docname="compass.svg"
   inkscape:version="1.2 (dc2aedaf03, 2022-05-15)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs362" />
  <sodipodi:namedview
     id="namedview360"
     pagecolor="#505050"
     bordercolor="#ffffff"
     borderopacity="1"
     inkscape:showpageshadow="0"
     inkscape:pageopacity="0"
     inkscape:pagecheckerboard="1"
     inkscape:deskcolor="#505050"
     showgrid="false"
     inkscape:zoom="1.9728279"
     inkscape:cx="78.56742"
     inkscape:cy="120.63901"
     inkscape:window-width="1920"
     inkscape:window-height="1017"
     inkscape:window-x="-8"
     inkscape:window-y="-8"
     inkscape:window-maximized="1"
     inkscape:current-layer="compass" />
  <text
     x="113"
     y="29.316406"
     font-size="20px"
     font-family="Helvetica"
     fill="#ffffff"
     id="text183"
     style="-inkscape-font-specification:'Arial, Bold';font-family:Arial;font-weight:bold;font-style:normal;font-stretch:normal;font-variant:normal;font-size:20px;font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;fill:#ffffff">N</text>
  <text
     x="213.41797"
     y="128"
     font-size="20px"
     font-family="Helvetica"
     fill="#ffffff"
     id="text185"
     style="-inkscape-font-specification:'Arial, Bold';font-family:Arial;font-weight:bold;font-style:normal;font-stretch:normal;font-variant:normal;font-size:20px;font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;fill:#ffffff">E</text>
  <text
     x="113"
     y="224.56055"
     font-size="20px"
     font-family="Helvetica"
     fill="#ffffff"
     id="text187"
     style="-inkscape-font-specification:'Arial, Bold';font-family:Arial;font-weight:bold;font-style:normal;font-stretch:normal;font-variant:normal;font-size:20px;font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;fill:#ffffff">S</text>
  <text
     x="14.755859"
     y="128"
     font-size="20px"
     font-family="Helvetica"
     fill="#ffffff"
     id="text189"
     style="-inkscape-font-specification:'Arial, Bold';font-family:Arial;font-weight:bold;font-style:normal;font-stretch:normal;font-variant:normal;font-size:20px;font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;fill:#ffffff">W</text>
  <line
     x1="120"
     y1="0"
     x2="120"
     y2="10"
     stroke="#ffffff"
     stroke-width="2"
     id="line191" />
  <line
     x1="130.45869"
     y1="0.45663643"
     x2="129.58713"
     y2="10.418583"
     stroke="#808080"
     stroke-width="1"
     id="line195" />
  <line
     x1="140.83778"
     y1="1.8230705"
     x2="139.1013"
     y2="11.671146"
     stroke="#808080"
     stroke-width="1"
     id="line197" />
  <line
     x1="151.05829"
     y1="4.0889015"
     x2="148.47009"
     y2="13.748158"
     stroke="#808080"
     stroke-width="1"
     id="line199" />
  <line
     x1="161.04242"
     y1="7.2368851"
     x2="157.62222"
     y2="16.633812"
     stroke="#808080"
     stroke-width="1"
     id="line201" />
  <line
     x1="170.71419"
     y1="11.243065"
     x2="166.48801"
     y2="20.306145"
     stroke="#808080"
     stroke-width="1"
     id="line203" />
  <line
     x1="180"
     y1="16.07695"
     x2="175"
     y2="24.737206"
     stroke="#ffffff"
     stroke-width="2"
     id="line205" />
  <line
     x1="188.82918"
     y1="21.701756"
     x2="183.09341"
     y2="29.893276"
     stroke="#808080"
     stroke-width="1"
     id="line209" />
  <line
     x1="197.13451"
     y1="28.074665"
     x2="190.70663"
     y2="35.735115"
     stroke="#808080"
     stroke-width="1"
     id="line211" />
  <line
     x1="204.85281"
     y1="35.147186"
     x2="197.78175"
     y2="42.218254"
     stroke="#808080"
     stroke-width="1"
     id="line213" />
  <line
     x1="211.92534"
     y1="42.865486"
     x2="204.26489"
     y2="49.293365"
     stroke="#808080"
     stroke-width="1"
     id="line215" />
  <line
     x1="218.29825"
     y1="51.17083"
     x2="210.10672"
     y2="56.906593"
     stroke="#808080"
     stroke-width="1"
     id="line217" />
  <line
     x1="223.92305"
     y1="60"
     x2="215.26279"
     y2="65"
     stroke="#ffffff"
     stroke-width="2"
     id="line219" />
  <line
     x1="228.75693"
     y1="69.285812"
     x2="219.69386"
     y2="73.511993"
     stroke="#808080"
     stroke-width="1"
     id="line223" />
  <line
     x1="232.76311"
     y1="78.957581"
     x2="223.3662"
     y2="82.377777"
     stroke="#808080"
     stroke-width="1"
     id="line225" />
  <line
     x1="235.9111"
     y1="88.941711"
     x2="226.25185"
     y2="91.529907"
     stroke="#808080"
     stroke-width="1"
     id="line227" />
  <line
     x1="238.17693"
     y1="99.162216"
     x2="228.32886"
     y2="100.8987"
     stroke="#808080"
     stroke-width="1"
     id="line229" />
  <line
     x1="239.54337"
     y1="109.54131"
     x2="229.58142"
     y2="110.41287"
     stroke="#808080"
     stroke-width="1"
     id="line231" />
  <line
     x1="240"
     y1="120"
     x2="230"
     y2="120"
     stroke="#ffffff"
     stroke-width="2"
     id="line233" />
  <line
     x1="239.54337"
     y1="130.45869"
     x2="229.58142"
     y2="129.58713"
     stroke="#808080"
     stroke-width="1"
     id="line237" />
  <line
     x1="238.17693"
     y1="140.83778"
     x2="228.32886"
     y2="139.1013"
     stroke="#808080"
     stroke-width="1"
     id="line239" />
  <line
     x1="235.9111"
     y1="151.05829"
     x2="226.25185"
     y2="148.47009"
     stroke="#808080"
     stroke-width="1"
     id="line241" />
  <line
     x1="232.76311"
     y1="161.04242"
     x2="223.3662"
     y2="157.62222"
     stroke="#808080"
     stroke-width="1"
     id="line243" />
  <line
     x1="228.75693"
     y1="170.71419"
     x2="219.69386"
     y2="166.48801"
     stroke="#808080"
     stroke-width="1"
     id="line245" />
  <line
     x1="223.92305"
     y1="180"
     x2="215.26279"
     y2="175"
     stroke="#ffffff"
     stroke-width="2"
     id="line247" />
  <line
     x1="218.29825"
     y1="188.82918"
     x2="210.10672"
     y2="183.09341"
     stroke="#808080"
     stroke-width="1"
     id="line251" />
  <line
     x1="211.92534"
     y1="197.13451"
     x2="204.26489"
     y2="190.70663"
     stroke="#808080"
     stroke-width="1"
     id="line253" />
  <line
     x1="204.85281"
     y1="204.85281"
     x2="197.78175"
     y2="197.78175"
     stroke="#808080"
     stroke-width="1"
     id="line255" />
  <line
     x1="197.13451"
     y1="211.92534"
     x2="190.70663"
     y2="204.26489"
     stroke="#808080"
     stroke-width="1"
     id="line257" />
  <line
     x1="188.82918"
     y1="218.29825"
     x2="183.09341"
     y2="210.10672"
     stroke="#808080"
     stroke-width="1"
     id="line259" />
  <line
     x1="180"
     y1="223.92305"
     x2="175"
     y2="215.26279"
     stroke="#ffffff"
     stroke-width="2"
     id="line261" />
  <line
     x1="170.71419"
     y1="228.75693"
     x2="166.48801"
     y2="219.69386"
     stroke="#808080"
     stroke-width="1"
     id="line265" />
  <line
     x1="161.04242"
     y1="232.76312"
     x2="157.62222"
     y2="223.3662"
     stroke="#808080"
     stroke-width="1"
     id="line267" />
  <line
     x1="151.05829"
     y1="235.9111"
     x2="148.47009"
     y2="226.25183"
     stroke="#808080"
     stroke-width="1"
     id="line269" />
  <line
     x1="140.83778"
     y1="238.17694"
     x2="139.1013"
     y2="228.32886"
     stroke="#808080"
     stroke-width="1"
     id="line271" />
  <line
     x1="130.45869"
     y1="239.54337"
     x2="129.58713"
     y2="229.58142"
     stroke="#808080"
     stroke-width="1"
     id="line273" />
  <line
     x1="120"
     y1="240"
     x2="120"
     y2="230"
     stroke="#ffffff"
     stroke-width="2"
     id="line275" />
  <line
     x1="109.54131"
     y1="239.54337"
     x2="110.41286"
     y2="229.58142"
     stroke="#808080"
     stroke-width="1"
     id="line279" />
  <line
     x1="99.162216"
     y1="238.17694"
     x2="100.8987"
     y2="228.32886"
     stroke="#808080"
     stroke-width="1"
     id="line281" />
  <line
     x1="88.941711"
     y1="235.9111"
     x2="91.529907"
     y2="226.25183"
     stroke="#808080"
     stroke-width="1"
     id="line283" />
  <line
     x1="78.957581"
     y1="232.76312"
     x2="82.377785"
     y2="223.3662"
     stroke="#808080"
     stroke-width="1"
     id="line285" />
  <line
     x1="69.285812"
     y1="228.75693"
     x2="73.511993"
     y2="219.69386"
     stroke="#808080"
     stroke-width="1"
     id="line287" />
  <line
     x1="60"
     y1="223.92305"
     x2="65"
     y2="215.26279"
     stroke="#ffffff"
     stroke-width="2"
     id="line289" />
  <line
     x1="51.170826"
     y1="218.29825"
     x2="56.906593"
     y2="210.10672"
     stroke="#808080"
     stroke-width="1"
     id="line293" />
  <line
     x1="42.865486"
     y1="211.92534"
     x2="49.293362"
     y2="204.26489"
     stroke="#808080"
     stroke-width="1"
     id="line295" />
  <line
     x1="35.147186"
     y1="204.85281"
     x2="42.218254"
     y2="197.78175"
     stroke="#808080"
     stroke-width="1"
     id="line297" />
  <line
     x1="28.074667"
     y1="197.13451"
     x2="35.735111"
     y2="190.70663"
     stroke="#808080"
     stroke-width="1"
     id="line299" />
  <line
     x1="21.701756"
     y1="188.82918"
     x2="29.893274"
     y2="183.09341"
     stroke="#808080"
     stroke-width="1"
     id="line301" />
  <line
     x1="16.076952"
     y1="180"
     x2="24.737206"
     y2="175"
     stroke="#ffffff"
     stroke-width="2"
     id="line303" />
  <line
     x1="11.243066"
     y1="170.71419"
     x2="20.306143"
     y2="166.48801"
     stroke="#808080"
     stroke-width="1"
     id="line307" />
  <line
     x1="7.2368855"
     y1="161.04242"
     x2="16.633812"
     y2="157.62222"
     stroke="#808080"
     stroke-width="1"
     id="line309" />
  <line
     x1="4.088901"
     y1="151.05829"
     x2="13.748159"
     y2="148.47009"
     stroke="#808080"
     stroke-width="1"
     id="line311" />
  <line
     x1="1.8230697"
     y1="140.83778"
     x2="11.671147"
     y2="139.1013"
     stroke="#808080"
     stroke-width="1"
     id="line313" />
  <line
     x1="0.45663622"
     y1="130.45869"
     x2="10.418583"
     y2="129.58713"
     stroke="#808080"
     stroke-width="1"
     id="line315" />
  <line
     x1="0"
     y1="120"
     x2="10"
     y2="120"
     stroke="#ffffff"
     stroke-width="2"
     id="line317" />
  <line
     x1="0.45663622"
     y1="109.54131"
     x2="10.418583"
     y2="110.41287"
     stroke="#808080"
     stroke-width="1"
     id="line321" />
  <line
     x1="1.8230697"
     y1="99.162216"
     x2="11.671147"
     y2="100.8987"
     stroke="#808080"
     stroke-width="1"
     id="line323" />
  <line
     x1="4.088901"
     y1="88.941711"
     x2="13.748159"
     y2="91.529907"
     stroke="#808080"
     stroke-width="1"
     id="line325" />
  <line
     x1="7.2368855"
     y1="78.957581"
     x2="16.633812"
     y2="82.377777"
     stroke="#808080"
     stroke-width="1"
     id="line327" />
  <line
     x1="11.243066"
     y1="69.285812"
     x2="20.306143"
     y2="73.511993"
     stroke="#808080"
     stroke-width="1"
     id="line329" />
  <line
     x1="16.076952"
     y1="60"
     x2="24.737206"
     y2="65"
     stroke="#ffffff"
     stroke-width="2"
     id="line331" />
  <line
     x1="21.701756"
     y1="51.17083"
     x2="29.893274"
     y2="56.906593"
     stroke="#808080"
     stroke-width="1"
     id="line335" />
  <line
     x1="28.074667"
     y1="42.865486"
     x2="35.735111"
     y2="49.293365"
     stroke="#808080"
     stroke-width="1"
     id="line337" />
  <line
     x1="35.147186"
     y1="35.147186"
     x2="42.218254"
     y2="42.218254"
     stroke="#808080"
     stroke-width="1"
     id="line339" />
  <line
     x1="42.865486"
     y1="28.074665"
     x2="49.293362"
     y2="35.735115"
     stroke="#808080"
     stroke-width="1"
     id="line341" />
  <line
     x1="51.170826"
     y1="21.701756"
     x2="56.906593"
     y2="29.893276"
     stroke="#808080"
     stroke-width="1"
     id="line343" />
  <line
     x1="60"
     y1="16.07695"
     x2="65"
     y2="24.737206"
     stroke="#ffffff"
     stroke-width="2"
     id="line345" />
  <line
     x1="69.285812"
     y1="11.243065"
     x2="73.511993"
     y2="20.306145"
     stroke="#808080"
     stroke-width="1"
     id="line349" />
  <line
     x1="78.957581"
     y1="7.2368851"
     x2="82.377785"
     y2="16.633812"
     stroke="#808080"
     stroke-width="1"
     id="line351" />
  <line
     x1="88.941711"
     y1="4.0889015"
     x2="91.529907"
     y2="13.748158"
     stroke="#808080"
     stroke-width="1"
     id="line353" />
  <line
     x1="99.162216"
     y1="1.8230705"
     x2="100.8987"
     y2="11.671146"
     stroke="#808080"
     stroke-width="1"
     id="line355" />
  <line
     x1="109.54131"
     y1="0.45663643"
     x2="110.41286"
     y2="10.418583"
     stroke="#808080"
     stroke-width="1"
     id="line357" />
</svg>
