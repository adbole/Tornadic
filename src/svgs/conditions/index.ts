export { ReactComponent as Drizzle } from "./cloud-drizzle-fill.svg";
export { ReactComponent as Fog } from "./cloud-fog-fill.svg";
export { ReactComponent as Lightning } from "./cloud-lightning-rain-fill.svg";
export { ReactComponent as CloudMoon } from "./cloud-moon-fill.svg";
export { ReactComponent as Rain } from "./cloud-rain-fill.svg";
export { ReactComponent as Cloud } from "./cloud-single-fill.svg";
export { ReactComponent as CloudSun } from "./cloud-sun-fill.svg";
export { ReactComponent as Clouds } from "./clouds-fill.svg";
export { ReactComponent as Moon } from "./moon-fill.svg";
export { ReactComponent as Snow } from "./snow.svg";
export { ReactComponent as Sun } from "./sun-fill.svg";