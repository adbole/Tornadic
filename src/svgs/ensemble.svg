<svg fill="currentColor" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
 <path d="m15.314 5.1134c0.21387 0.175 0.24522 0.49031 0.07 0.704l-4.5 5.5c-0.18727 0.22858-0.53086 0.24576-0.74 0.037l-2.587-2.587-3.656 5.027c-0.39701 0.50483-1.1663-0.05497-0.808-0.588l4-5.5c0.18054-0.2487 0.54059-0.2772 0.758-0.06l2.609 2.61 4.15-5.073c0.175-0.21387 0.49031-0.24522 0.704-0.07m-1-4c0.21387 0.175 0.24522 0.49031 0.07 0.704l-4.5 5.5c-0.18727 0.22858-0.53086 0.24576-0.74 0.037l-2.587-2.587-3.656 5.027c-0.39701 0.50483-1.1663-0.05497-0.808-0.588l4-5.5c0.18054-0.2487 0.54059-0.2772 0.758-0.06l2.609 2.61 4.15-5.073c0.175-0.21387 0.49031-0.24522 0.704-0.07m-14.314-1.1134h1v15h15v1h-16zm14.817 3.113c0.21387 0.175 0.24522 0.49031 0.07 0.704l-4.5 5.5c-0.18727 0.22858-0.53086 0.24576-0.74 0.037l-2.587-2.587-3.656 5.027c-0.39701 0.50483-1.1663-0.05497-0.808-0.588l4-5.5c0.18054-0.2487 0.54059-0.2772 0.758-0.06l2.609 2.61 4.15-5.073c0.175-0.21387 0.49031-0.24522 0.704-0.07"/>
</svg>
