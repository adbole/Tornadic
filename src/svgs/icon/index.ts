/// <reference types="vite-plugin-svgr/client" />

export { default as Tornadic } from "./tornadic.svg?react";
export { default as TornadicFull } from "./tornadic-full.svg?react";
