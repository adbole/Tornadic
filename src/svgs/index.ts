/// <reference types="vite-plugin-svgr/client" />

export { default as Ensemble } from "./ensemble.svg?react";
export { default as ExclamationTriangle } from "./exclamation-triangle-fill.svg?react";
export { default as Spinner } from "./spinner.svg?react";
export { default as WifiOff } from "./wifi-off.svg?react";
