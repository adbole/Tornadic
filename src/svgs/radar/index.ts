/// <reference types="vite-plugin-svgr/client" />

export { default as CircleSlashes } from "./circle-slashes.svg?react";
export { default as LocationDot } from "./current-location-dot.svg?react";
export { default as Cursor } from "./cursor-fill.svg?react";
export { default as Grid } from "./grid.svg?react";
export { default as Pause } from "./pause-fill.svg?react";
export { default as Play } from "./play-fill.svg?react";
