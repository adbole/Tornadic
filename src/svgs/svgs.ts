import {ReactComponent as Tornadic} from './tornadic.svg'
import { ReactComponent as Play } from './play-fill.svg'
import { ReactComponent as Pause } from './pause-fill.svg'
import { ReactComponent as Grid } from './grid.svg'
import { ReactComponent as Cursor } from './cursor-fill.svg'


export { 
    Tornadic,
    Play,
    Pause,
    Grid,
    Cursor
};