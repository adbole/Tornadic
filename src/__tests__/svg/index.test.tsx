import { render } from "@testing-library/react";

import * as svgs from "svgs";


test.each(Object.entries(svgs))("%s", (_, Svg) => {
    const { container } = render(<Svg />);

    expect(container).toMatchSnapshot();
});
