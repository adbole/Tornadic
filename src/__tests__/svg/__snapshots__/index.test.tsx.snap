// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ExclamationTriangle 1`] = `
<div>
  <svg
    fill="red"
    viewBox="0 0 16 16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M8.982 1.566a1.13 1.13 0 0 0-1.96 0L.165 13.233c-.457.778.091 1.767.98 1.767h13.713c.889 0 1.438-.99.98-1.767L8.982 1.566zM8 5c.535 0 .954.462.9.995l-.35 3.507a.552.552 0 0 1-1.1 0L7.1 5.995A.905.905 0 0 1 8 5zm.002 6a1 1 0 1 1 0 2 1 1 0 0 1 0-2z"
    />
  </svg>
</div>
`;

exports[`Spinner 1`] = `
<div>
  <svg
    stroke="currentcolor"
    viewBox="0 0 24 24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      style="transform-origin: center;"
    >
      <animatetransform
        attributeName="transform"
        attributeType="XML"
        dur="2s"
        from="0 0 0"
        repeatCount="indefinite"
        to="360 0 0"
        type="rotate"
      />
      <circle
        cx="12"
        cy="12"
        fill="none"
        r="9.5"
        stroke-linecap="round"
        stroke-width="3"
      >
        <animate
          attributeName="stroke-dasharray"
          dur="1.5s"
          keyTimes="0; 0.475; 1"
          repeatCount="indefinite"
          values="0 150; 42 150; 42 150"
        />
        <animate
          attributeName="stroke-dashoffset"
          dur="1.5s"
          keyTimes="0; 0.475; 1"
          repeatCount="indefinite"
          values="0; -16; -59"
        />
      </circle>
    </g>
  </svg>
</div>
`;

exports[`WifiOff 1`] = `
<div>
  <svg
    fill="currentColor"
    viewBox="0 0 16 16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M10.706 3.294A12.545 12.545 0 0 0 8 3C5.259 3 2.723 3.882.663 5.379a.485.485 0 0 0-.048.736.518.518 0 0 0 .668.05A11.448 11.448 0 0 1 8 4c.63 0 1.249.05 1.852.148l.854-.854zM8 6c-1.905 0-3.68.56-5.166 1.526a.48.48 0 0 0-.063.745.525.525 0 0 0 .652.065 8.448 8.448 0 0 1 3.51-1.27L8 6zm2.596 1.404.785-.785c.63.24 1.227.545 1.785.907a.482.482 0 0 1 .063.745.525.525 0 0 1-.652.065 8.462 8.462 0 0 0-1.98-.932zM8 10l.933-.933a6.455 6.455 0 0 1 2.013.637c.285.145.326.524.1.75l-.015.015a.532.532 0 0 1-.611.09A5.478 5.478 0 0 0 8 10zm4.905-4.905.747-.747c.59.3 1.153.645 1.685 1.03a.485.485 0 0 1 .047.737.518.518 0 0 1-.668.05 11.493 11.493 0 0 0-1.811-1.07zM9.02 11.78c.238.14.236.464.04.66l-.707.706a.5.5 0 0 1-.707 0l-.707-.707c-.195-.195-.197-.518.04-.66A1.99 1.99 0 0 1 8 11.5c.374 0 .723.102 1.021.28zm4.355-9.905a.53.53 0 0 1 .75.75l-10.75 10.75a.53.53 0 0 1-.75-.75l10.75-10.75z"
    />
  </svg>
</div>
`;
